packages/main/src/Menu.ts

```diff
@@ -459,13 +459,14 @@ class Menu extends UI5Element {
 		});
 	}
 
-	_createSubMenu(item: MenuItem, openerId: string) {
+	_createSubMenu(item: MenuItem, opener: HTMLElement, openerId: string) {
```

Comment:
Is it necessary to have both `opener` and `openerId`? If the second is always `opener.id` or `opener._id` or something like that that can always be derived and can't be anything else, please remove the third parameter.

Reply:
getting the `id` attribute of a DOM element that you already have does not make the browser query the DOM (what f.e. `querySelector` would). I don't know the code but if indeed `openerId` is just `opener.id`, then the third argument is not needed and there will be no performance penalty to do so. If truly a DOM operation would be needed (calling a browser API) then let's keep it as it is.

Reply:
I was left with the impression while debugging that the `openerId` is actually the inner `li` whereas the opener is a `ui5-menu-li` element. Those are referring the same element so you're right.